### PR TITLE
Changed the negatives from parenthesis to minus symbols

### PR DESCRIPTION
**Pull request by moneytree-doug**

### CODE REVIEW

**Comment by moneytree-doug** [comment]
@makdad

File: dist/isoCurrency.js [hunk 1]
```diff
@@ -894,8 +894,16 @@ angular.module('isoCurrency', ['isoCurrency.common'])
 			if (!currency) {
 				return amount;			
 			}
+
 			var fractionSize = (fraction === void 0) ? currency.fraction : fraction;
-			return $filter('currency')(amount, currency.symbol, fractionSize);
+	        var currencyFilter = $filter('currency');         
+
+	        if(amount < 0){
```
Comment (makdad) [inline dist/isoCurrency.js]: Comment would be good here.  Particularly if there's a design doc or someone that said "it should be this way", definitely write it down.  This is the kind of shit that someone says later" this is wrong" and we need some accountability of why it is this way

Comment (moneytree-doug) [inline dist/isoCurrency.js]: Accountability for why we should use negatives over parenthesis?

Comment (moneytree-doug) [inline dist/isoCurrency.js]: But to answer you, Angular uses parenthesis by default which is less understandable than the negative symbol based on the feedback I have received so far. It is just too technical for the masses.

Comment (makdad) [inline dist/isoCurrency.js]: Yes, leave that as a comment then.  The thing is, it's a design decision, not a technical implementation.  We don't have a design doc to point to and say "this is what it is".  It's up to your gut (which you clearly have an opinion on here).  That said, 6 months from now when 1 Guest tells us "hey it should be parenths" or whatever, that comment will remind me, or Ross, why that decision was made in the first place.

Startup lesson - Remaking and unmaking decisions that have already been made is one of the most expensive things you can do in a startup.

Comment (moneytree-doug) [inline dist/isoCurrency.js]: @makdad Okay, I just wanted to hear your reasoning even though it was trivial to add the comment. I appreciate the explanation.

**Comment by moneytree-doug** [comment]
@makdad

**Pull request merged**